src/app/core/store/documents/document.model.ts

```diff
@@ -21,9 +21,13 @@ export interface DocumentModel {
 
   id?: string;
   collectionCode: string;
+  collectionIcon?: string;
+  collectionColor?: string;
+  collectionName?: string;
```

Comment:
Is this really needed? Isn't there any other way how to access this collection information in the template without storing them in document model? I think there should be. But if it's too complicated and the code is a mess after such a change then we should keep the whole collection object here rather than just some of its properties.